package.json

```diff
@@ -32,16 +32,18 @@
   },
   "homepage": "http://gpii.net",
   "dependencies": {
+    "ajv": "^1.4.9",
+    "dedupe-infusion": "1.0.0",
     "first-discovery": "git+https://github.com/GPII/first-discovery.git",
     "gpii-express": "git+https://github.com/GPII/gpii-express.git",
-    "infusion": "2.0.0-dev.20151007T134312Z.8c33cd4",
-     "dedupe-infusion": "1.0.0"
+    "infusion": "git://github.com/amb26/infusion.git#d9ab2cea604cc3108bdf028d70874d339df22720",
+    "kettle": "git://github.com/amb26/kettle.git#KETTLE-32"
```

Comment:
do you have a recommendation on which revision to take for the branch, or i could just grab the last commit here.